### ISSUE TITLE
NodeJS v6 error - Cannot find module '../../infobox/infobox'

### ISSUE THREAD

**Issue by lazharichir** [spencermountain/wtf_wikipedia#98]
I am using wtf_wikipedia in Google Cloud Functions and they un NodeJS 6.x. At deployment, it triggers the below error, not sure if you could fix it so it can work on v6, too.

```
Code in file index.js can't be loaded.
Did you list all required modules in the package.json dependencies?
Detailed stack trace: Error: Cannot find module '../../infobox/infobox'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/user_code/node_modules/wtf_wikipedia/src/document/templates/index.js:6:17)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)"  
```

NODEJS: 6.11.5
WTF_WIKIPEDIA: 3.1.0

**Comment by spencermountain** [comment]
thanks Lazhar, that's helpful. will check it out today.

**Comment by spencermountain** [comment]
hey Lazhar, fixed this in `3.1.1` -looks like i title-cased the file, and linux isn't as forgiving as mac is about that.
thanks!
lemme know if you see something else.

**Comment by lazharichir** [comment]
will do, thanks a lot for the (VERY!) quick fix :)
